test/bin/mdx-test/failure/in-toplevel/test-case.md.expected

```diff
@@ -1,2 +1,2 @@
-Error in the OCaml file include code block in test-case.md at line 4:
+File "test-case.md", lines 3-4, characters 36-0: Error in the OCaml file include code block
```

Comment:
Weird character count `0` in the message, no idea where it comes from, since all the locs come from the parser.

Reply:
Removed the columns from the pretty-printer